src/coreclr/inc/CMakeLists.txt

```diff
@@ -58,13 +58,13 @@ if(FEATURE_JIT_PITCHING)
 endif(FEATURE_JIT_PITCHING)
 
 # Compile *_i.cpp to lib
-_add_library(corguids_obj OBJECT ${CORGUIDS_SOURCES})
+add_library_clr(corguids_obj OBJECT ${CORGUIDS_SOURCES})
 add_library(corguids INTERFACE)
 target_sources(corguids INTERFACE $<TARGET_OBJECTS:corguids_obj>)
 
 # Binplace the inc files for packaging later.
 
-_install (FILES cfi.h
```

Comment:
Do file installs not need a component? Is this under `unknown`?

Reply:
They'll fall under the `coreclr_misc` subset and be installed when any component (other that iltools or jit) is built.